doc/release-process.md

```diff
@@ -53,7 +53,9 @@ In `configure.ac` and `clientversion.h`:
 If this release changes the behavior of the protocol or fixes a serious bug, we may
 also wish to change the `PROTOCOL_VERSION` in `version.h`.
 
-Build and commit to update versions, and then perform the following command:
+Build, then commit these changes to update versions.
```

Comment:
Needs to be "commit then build" so that the built binary doesn't include `-dirty`.